IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj" Name="Attenuation" UniqueId="0fc49883-5ab2-40aa-b640-e276e1d814ed" PrimaryDataType="IfcSoundPowerLevelMeasure" />
```

Comment:
We should look into existing noise emissions properties.

Reply:
Needs changing to IfcReal

Reply:
Done. c88812ba98efa9af1e4de31bf345870dedc28a76